### ISSUE TITLE
input with required property render a box-shadow border in Firefox

### ISSUE THREAD

**Issue by felquis** [react-toolbox/react-toolbox#916]
In `view-source:resource://gre-resources/forms.css` Firefox always set this property

```css
/**
 * Set default style for invalid elements.
 */
:not(output):-moz-ui-invalid {
  box-shadow: 0 0 1.5px 1px red;
}
```
![screenshot from 2016-10-31 13-56-11](https://cloud.githubusercontent.com/assets/736728/19861116/83da9756-9f72-11e6-883a-bfe5c821d175.png)

Which cause Firefox to render a red border, I fixed it in my project adding

```css
input[required]:-moz-ui-invalid {
  box-shadow: none;
}
```

Firefox after CSS Fix
![screenshot from 2016-10-31 13-52-46](https://cloud.githubusercontent.com/assets/736728/19861129/91d25452-9f72-11e6-91aa-4bd3b22b3321.png)

Chrome Before and After CSS fix
![screenshot from 2016-10-31 14-05-15](https://cloud.githubusercontent.com/assets/736728/19861307/3396d1a0-9f73-11e6-8b1a-15bb69aa7bfd.png)

**Comment by samsch** [comment]
@javivelasco Your fix doesn't work when not using commons.scss, such as when just importing the Input component.

**Comment by javivelasco** [comment]
Yes @samsch but that's a different issue. Currently there are some resets and fixed font sizes that needs to be added globally so if you include a component alone it's not enough. @okonet is working on a fix #1006
